reorganize.py

```diff
@@ -0,0 +1,90 @@
+#reorganize.py
+#takes the output of the outbreak simulator and turns it into a csv file with relevant data shown in tabular format
+
+import numpy as np
+import pandas as pd
+import re
+import matplotlib.pyplot as plt
+import argparse
+
+def identify_timestamps(dictionary):
+    timestamps = [float(x.rsplit('_', 1)[1]) for x in dictionary.keys()]
+    return timestamps
+
+def delete_extra(dictionary):
+    del_list = [x for x in dictionary.keys() if re.search(r'\d+', x) == None]
+    for i in del_list:
+        del dictionary[i]
+    name = [x.rsplit('_', 1)[0] for x in dictionary.keys()]
+
+    del_list_t = []
+    for x in name:
+        count = 0
+        for y in name:
+            if x == y:
+                count += 1
+        if count < 3:
+            del_list_t.append(x)

```

Comment:
have a look at `itertools` and `counter`. For this particular case, you might be able to do a list comprehension with condition `if name.count(x) >2`.